code/testsuite/base-xml.ftl

```diff
@@ -102,7 +102,7 @@
 		</skin>
 		<cr><@pcstring tag="CR"/></cr>
 		<face>
-		<face><@pcstring tag="FACE"/></face>
+			<face><#if pc.val.os_size!false == true><#if pc.val.face[0] == pc.val.face[1]>${pc.val.face[0]} ft.<#else>${pc.val.face[0]} ft. by test ${pc.val.face[1]} ft.</#if><#else>${pcstring('FACE')}</#if></face>
```

Comment:
It strikes me that instead of forcing you to have pc.val.os_size, which really replicates "do I have the FACE code control?" we should give you real access to the code control... wouldn't that make more sense (and save you a variable?)